src/utils/ThemeSlice/themeSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  currentTheme: ["light", "dark"].includes(localStorage.getItem("currentTheme"))
+    ? localStorage.getItem("currentTheme")
+    : "light",
```

Comment:
suggestion: this could be `currentTheme: localStorage.getItem("currentTheme") === "dark" ? "dark" : "light"`

Reply:
My intent was to add more themes going forward, so it'll not be binary.